gnomad_constraint/pipeline/constraint_pipeline.py

```diff
@@ -175,6 +177,29 @@ def main(args):
                 ).write(training_resources[region].path, overwrite=overwrite)
             logger.info("Done with creating training dataset.")
 
+        # Build plateau and coverage models for autosomes/pseudoautosomal regions,
+        # chromosome X, and chromosome Y
+        if args.build_and_apply_models:
```

Comment:
My solution to address the following comment is to combine the building model part and applying model part as what Konrad did in `gnomad_lof` and save the models as global annotations in the output tables of applying model part. We may be able to make the output of `build_models()` be a dictionary and save it as a json file, but I felt it's much more redundant compared to the current solution when converting data type between tuple, dictionary, and json format. Another reason is that without the output files of applying model part, we don't know how the models performed.

> the models get built but not written, right? should they get stored as a pickle file or in an ht's global fields so this code doesn't need to be run more than once?

Reply:
what if something crashes during only the apply models part of `build_and_apply_models`? you wouldn't want to rebuild the models for nothing. what about writing out the models as hailexpressions? https://hail.is/docs/0.2/experimental/index.html#hail.experimental.write_expression